### DISCUSSION TITLE
Allow switching between players in multi-spectator via clicking leaderboard cards

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#18315] [Ideas]
Hope this hasn't been suggested before. I don't know how "gaze" is translated correctly, but I hope that the section shown in the screenshot clearly shows what I want.
![osu_2022-05-17_19-49-50](https://user-images.githubusercontent.com/59089206/168886075-5dc83fec-34ed-4c1e-817c-841f87f3f4c2.jpg)

**Comment by bdach** [comment]
you can already kind of do this though. click on the spectator area again after zooming in on one player to zoom back out again.

unless you mean doing it with a *specific* player, instantly, without zoom-out, via clicking the scoreboard specifically?

**Reply by Moltanicaa** [reply]
Yea, specific, without zoom-out

**Comment by marioman63** [comment]
I don't use spectator mode that often, so when I tried this, I was surprised it didn't work. Makes perfect sense to do this IMO.
